packages/cozy-harvest-lib/test/index.spec.js

```diff
@@ -1,4 +1,4 @@
-describe('index', () => {
```

Comment:
Is it expected ? Why not simply remove the test altogether ? If this is expected and the file should stay, could you add a comment ?

Reply:
I'll remove it, it's not a very useful test anymore anyway.